Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -108,7 +114,10 @@ ExceptionOr<void> HTMLScriptElement::setTextContent(ExceptionOr<String> value)
     if (value.hasException())
         return value.releaseException();
 
-    setTextContent(value.releaseReturnValue());
+    auto newValue = value.releaseReturnValue();
+
+    m_scriptText = newValue;
+    setTextContent(WTFMove(newValue));
```

Comment:
I’m a little surprised that we set `m_scriptText` _exactly_ the same way when calling `setTextContent` or `setInnerText`. Do those two functions do exactly the same thing? If so, then why doesn’t one call the other? If not, then why does `m_scriptText` get set exactly the same?

Reply:
So I can answer the why. That's how it's specced and how Chromium has implemented it. As for whether they behave differently I'm not entirely sure, I know .text and .textContent are identical, .innerText has a more complicated spec but I'm not sure if those differences are meaningful wrt script elements. I'll make a spec issue to see if this is an oversight or not.

Reply:
Raised issue https://github.com/w3c/trusted-types/issues/517